internal/cmd/operator_uninstall.go

```diff
@@ -15,8 +15,26 @@ func newOperatorUninstallCmd(cfg *action.Configuration) *cobra.Command {
 
 	cmd := &cobra.Command{
 		Use:   "uninstall <operator>",
-		Short: "Uninstall an operator",
-		Args:  cobra.ExactArgs(1),
+		Short: "Uninstall an operator and operands",
+		Long: `Uninstall removes the subscription, operator and optionally operands managed by the operator as well as the relevant operatorgroup.
+
+Warning: this command permanently deletes objects from the cluster. Running uninstall concurrently with other operations
+could result in undefined behavior. 
+
+The uninstall command first checks to find the subscription associated with the operator. It then 
+lists all operands found throughout the cluster for the operator
+specified if one is found. Since the scope of an operator is restricted by
+its operator group, this search will include namespace-scoped operands from the
+operator group's target namespaces and all cluster-scoped operands. 
+
+The operand-deletion strategy is then considered if any operands are found on-cluster. One of cancel|ignore|delete. 
+By default, the strategy is "cancel", which means that if any operands are found when deleting the operator abort the uninstall without deleting anything. 
+The "ignore" strategy keeps the operands on cluster and deletes the subscription and the operator.
+The "delete" strategy deletes the subscription, operands, and after they have finished finalizing, the operator itself.
+
+Setting --delete-operator-groups to true will delete the operatorgroup in the provided namespace if no other active subscriptions 
+are currently in that namespace, after removing the operator. The subscription and operatorgroup will be removed even if the operator is not found.`,
```

Comment:
Should we have a consistent line break to make this format well on most consoles (perhaps cap it at 80 or 100 characters?)

This makes me wonder if there's a library that help us do this consistently across all commands.

Reply:
Per https://github.com/spf13/cobra/issues/484 looks like we should just use strings instead of string literals?

Reply:
I think the paragraphs and line-splitting with string literals like you have in the current long description are fine. My only complaint is that there does not seem to be a consistent line width.

I would have expected something like:
```suggestion
		Long: `Uninstall removes the subscription, operator and optionally operands managed by
the operator as well as the relevant operatorgroup.

Warning: this command permanently deletes objects from the cluster. Running
uninstall concurrently with other operations could result in undefined
behavior. 

The uninstall command first checks to find the subscription associated with the
operator. It then lists all operands found throughout the cluster for the
operator specified if one is found. Since the scope of an operator is restricted
by its operator group, this search will include namespace-scoped operands from
the operator group's target namespaces and all cluster-scoped operands. 

The operand-deletion strategy is then considered if any operands are found
on-cluster. One of cancel|ignore|delete. 
  - By default, the strategy is "cancel", which means that if any operands are
    found when deleting the operator abort the uninstall without deleting
    anything. 
  - The "ignore" strategy keeps the operands on cluster and deletes the
    subscription and the operator.
  - The "delete" strategy deletes the subscription, operands, and after they have
    finished finalizing, the operator itself.

Setting --delete-operator-groups to true will delete the operatorgroup in the
provided namespace if no other active subscriptions are currently in that
namespace, after removing the operator. The subscription and operatorgroup will
be removed even if the operator is not found.`,
```